moveit_plugins/moveit_simple_controller_manager/include/moveit_simple_controller_manager/action_based_controller_handle.h

```diff
@@ -56,9 +57,9 @@ class ActionBasedControllerHandleBase : public moveit_controller_manager::MoveIt
 
   virtual void addJoint(const std::string& name) = 0;
   virtual void getJoints(std::vector<std::string>& joints) = 0;
-  virtual void configure(XmlRpc::XmlRpcValue& /* config */)
-  {
-  }
+
+protected:
+  const rclcpp::Logger LOGGER = rclcpp::get_logger("action_based_controller_handle");
```

Comment:
Not sure if this's how we want to handle having the LOGGER in a header file

Reply:
You should be able to just declare it in the header and then define it in the cpp file.